docs/operator-manual/architecture.md

```diff
@@ -32,3 +32,4 @@ applications and compares the current, live state against the desired target sta
 the repo). It detects `OutOfSync` application state and optionally takes corrective action. It
 is responsible for invoking any user-defined hooks for lifecycle events (PreSync, Sync, PostSync)
 
+### ApplicationSet Controller
```

Comment:
Please add more information about the ApplicationSet Controller in the architecture section